recipes/bulker/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "bulker" %}
+{% set version = "0.7.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/databio/{{ name }}/archive/refs/tags//v{{ version }}.tar.gz
+  sha256: 93bd45c123271be9558a1f2d68624861e180fe7aa1aaf676db3457ba04b34937
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [not win and py>=36] - this package is an interface to Docker / Singularity containers, thus the [not win]
```

Comment:
To prevent installation on windows, you can add the `__unix` [virtual package](https://docs.conda.io/projects/conda/en/stable/user-guide/tasks/manage-virtual.html#managing-virtual-packages) as run dependency.
You can then make the package [noarch: python](https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python) instead and add a minimum version on python in host and run requirements: `python >=3.6`.